# baumer_neoapi
node_baumer_ir_cam:
  ros__parameters:
    topic_out_raw_image:        "/Baumer_ir/raw_image"
    FrameRate:                  30.0
    ExposureAuto:               "Continuous"       # Options: On, Off, Once, Continuous
    WhiteBalance:               "Manual"    # Options: Auto, Manual
    WhiteBalanceRed:            4500
    WhiteBalanceBlue:           3000  
    TriggerMode:                "On"
    TriggerSource:              "Software"
    ExposureTime:               20000
    Gain:                       1.2
    Width:                      300
    Height:                     300
    OffsetX:                    800
    OffsetY:                    100

# optris:

# webcam:

# microphone: